pkg/controller/garbagecollector/garbagecollector_test.go

```diff
@@ -856,7 +856,17 @@ func TestGarbageCollectorSync(t *testing.T) {
 	stopCh := make(chan struct{})
 	defer close(stopCh)
 	go gc.Run(1, stopCh)
-	go gc.Sync(fakeDiscoveryClient, 10*time.Millisecond, stopCh)
+	// The pseudo-code of GarbageCollector.Sync():
+	// GarbageCollector.Sync():
+	//    wait.Until() loops with `period` until the `stopCh` is closed :
+	//        wait.PollImmediateUntil() loops with 100ms (hardcode) util the `stopCh` is closed:
+	//            controller.WaitForCacheSync() loops with `syncedPollPeriod` (hardcoded to 100ms),
+	//            until either its stop channel is closed after `period`, or all caches synced.
+	//
+	// 200ms of period makes ~2 executions of controller.WaitForCacheSync() in every wait.PollImmediateUntil() loop.
```

Comment:
This line isn't accurate. Every loop in wait.PollImmediateUntil() always executes controller.WaitForCacheSync() once. 

Can you update it to "Setting the period to 200ms allows the WaitForCacheSync() to check for cache sync ~2 times in every wait.PollImmediateUntil() loop."